fuzz_plugin.py

```diff
@@ -70,16 +71,22 @@ def fuzz_plugin(
     else:
         enabled_features = enabled_features.split(",")
 
-    assert all(
-        [letter in string.ascii_letters + string.digits + "-_" for letter in slug]
-    )
-
-    plugin_info_dict = None
-    print("Looking for", slug)
-    plugin_info_dict = requests.get(
-        f"https://api.wordpress.org/plugins/info/1.2/?action=plugin_information"
-        f"&request[slug]={slug}"
-    ).json()
+    if slug.endswith('.zip') and os.path.exists(slug):
+        plugin_path = slug
+        slug = os.path.basename(slug)
+        plugin_info_dict = {'version': 0, 'active_installs': 0, 'sections': {'description': ''}, 'from_file': True}
+    else:
+        assert all(
+            [letter in string.ascii_letters + string.digits + "-_" for letter in slug]
+        )
+
+        plugin_info_dict = None
+        print("Looking for", slug)
+        plugin_info_dict = requests.get(
+            f"https://api.wordpress.org/plugins/info/1.2/?action=plugin_information"
+            f"&request[slug]={slug}"
+        ).json()
+        plugin_info_dict['from_file'] = False
```

Comment:
[nitpicking] plugin_info_dict has two purposes - contain the plugin information and information how we're installing it - what do you think about storing from_file in a separate variable?